<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Foreground</class>
 <widget class="QWidget" name="Foreground">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>938</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#Foreground {
    border-image: url(:/images/pics/bg.png) 0 0 0 0 stretch;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="title">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>83</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>83</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/images/pics/usc_logo.png</pixmap>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QFrame" name="fglabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>780</width>
         <height>111</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>780</width>
         <height>111</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#fglabel { 
   background-color: #0BC200;
	border-radius: 9px;
    border-bottom: 2px solid #020202;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
         <widget class="QLabel" name="fgtext">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>762</width>
            <height>90</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>762</width>
            <height>90</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#fgtext {
   color: #FFF;
   text-align: center;
   font-family: &quot;Roboto Condensed&quot;;
   font-style: italic;
   font-weight: 700;
   font-size: 50px;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
          <property name="text">
           <string>Select Foreground Template</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QPushButton" name="image2">
       <property name="minimumSize">
        <size>
         <width>425</width>
         <height>305</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>425</width>
         <height>305</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#image2{
    border: none; 
    background-image: url(:/images/pics/2.png); 
    background-repeat: no-repeat;
    background-position: center;
	background-size: contain;
	background-color: transparent;
	width: 241px;
   height: 141px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="image6">
       <property name="minimumSize">
        <size>
         <width>425</width>
         <height>305</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>425</width>
         <height>305</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#image6{
    border: none;
    background-image: url(:/images/pics/6.png);
    background-repeat: no-repeat;
    background-position: center;
	background-size: contain;
	background-color: transparent;
	width: 241px;
    height: 131px;
	padding: 10px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="image5">
       <property name="minimumSize">
        <size>
         <width>425</width>
         <height>305</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>425</width>
         <height>305</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#image5{
    border: none; 
    background-image: url(:/images/pics/5.png); 
    background-repeat: no-repeat;
    background-position: center;
	background-size: contain;
	background-color: transparent;
	width: 241px;
   height: 141px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="image1">
       <property name="minimumSize">
        <size>
         <width>425</width>
         <height>305</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>425</width>
         <height>305</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">#image1{
    border: none; 
    background-image: url(:/images/pics/1.png); 
    background-repeat: no-repeat;
    background-position: center;
	background-size: contain;
	background-color: transparent;
	width: 241px;
   height: 141px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="image4">
       <property name="minimumSize">
        <size>
         <width>425</width>
         <height>305</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>425</width>
         <height>305</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#image4{
    border: none; 
    background-image: url(:/images/pics/4.png); 
    background-repeat: no-repeat;
    background-position: center;
	background-size: contain;
	background-color: transparent;
	width: 241px;
   height: 141px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="image3">
       <property name="minimumSize">
        <size>
         <width>425</width>
         <height>305</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>425</width>
         <height>305</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#image3{
    border: none; 
    background-image: url(:/images/pics/3.png); 
    background-repeat: no-repeat;
    background-position: center;
	background-size: contain;
	background-color: transparent;
	width: 241px;
   height: 141px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="back">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>248</width>
         <height>68</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>248</width>
         <height>68</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#back{
	border-radius: 9px;
	border-bottom: 2px solid #020202;
	background: #0BC200;
	color: #FFF;
   text-align: center;
   font-family: &quot;Roboto Condensed&quot;;
   font-style: italic;
   font-weight: 500;
   font-size: 30px;
}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="confirm">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>248</width>
         <height>68</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>248</width>
         <height>68</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#confirm{
	border-radius: 9px;
	border-bottom: 2px solid #020202;
	background: #414141;
	color: #000;
	text-align: center;
	font-family: &quot;Roboto Condensed&quot;;
	font-style: italic;
	font-weight: 500;
	font-size: 30px;
}</string>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
